.rubocop.yml

```diff
@@ -1361,9 +1361,6 @@ Style/FormatString:
 Style/FormatStringToken:
   Enabled: false
 
-Style/FrozenStringLiteralComment:
-  Enabled: false
-
```

Comment:
`frozen_string_literal` was introduced on 2015-12-15 and the RuboCop style cop to enforce its usage with a `true` value was introduced soon after in early 2016. By simply no longer disabling the cop, we'll benefit from the default behavior of it enforcing the use of `frozen_string_literal: true`. Party like it's 2016!